src/reference/snakemake-style-guide.rst

```diff
@@ -46,3 +46,143 @@ Avoid the ``message`` rule attribute
 -  When the ``message`` attribute is defined, Snakemake suppresses other
    critical details that otherwise get displayed by default (e.g., job
    id, rule name, input, output, etc.).
+
+Use a config.yaml file
+======================
+
+Configuration is data and should live inside a YAML file named
+``config.yaml``. You can access it in your Snakefile by including the
+line:
+
+::
+
+   configfile: "config.yaml"
+
+and then using the ``config`` dictionary provided in scope afterwards.
+
+Allow for easy local config overrides
+=====================================
+
+By including the following snippet in your Snakefile, you can allow for
+optional, local configuration overrides which are more persistent than
+using ``--config`` options to ``snakemake``.
+
+::
+
+   from pathlib import Path
+
+   if Path("config_local.yaml").is_file():
+       configfile: "config_local.yaml"
+
+If you do this, you should also add a ``/config_local.yaml`` line to
+your repository's top-level ``.gitignore`` file.
```

Comment:
This should also probably be dropped.

Reply:
Maybe replace with the current standard that we use in [monkeypox](https://github.com/nextstrain/monkeypox/blob/5c461dc7e90cd70c1f16b193f82fd1666d4c95e2/Snakefile#L20-L22):

```python
if not config:
    configfile: "config/config_hmpxv1.yaml"
```

Reply:
Done: b0c50da30f3e55e099199cc13e806bf12a5e0015